src/subscription_manager/managercli.py

```diff
@@ -1452,6 +1452,14 @@ def _do_command(self):
             # TODO: grab the list of valid options, and check
             self.cp.updateConsumer(consumer['uuid'], release=self.options.release)
 
+        # Sync the syspurpose cache
+        try:
+            store = syspurposelib.get_sys_purpose_store()
+            if store:
+                store.sync()
```

Comment:
This add two REST API calls to registration process and none of these REST API calls is necessary, because we already received necessary information in previous calls to candlepin server. New REST API calls:

 - GET /candlepin/status
 - GET /candlepin/consumers/<consumer_uuid>

Problem is that `store` is instance of `SyncedStore` and it is implemented in `./syspurpose/src/files.py` and this code does not have access to caching capabilities implemented in `./src/subscription_manager` package.